server/datastore/mysql/apple_mdm.go

```diff
@@ -5372,3 +5377,31 @@ func (ds *Datastore) CountABMTokensWithTermsExpired(ctx context.Context) (int, e
 	}
 	return count, nil
 }
+
+func (ds *Datastore) GetABMTokenOrgNamesForTeam(ctx context.Context, teamID *uint) ([]string, error) {
+	stmt := `
+SELECT DISTINCT
+	abmt.organization_name
+FROM
+	abm_tokens abmt
+	JOIN host_dep_assignments hda ON hda.abm_token_id = abmt.id
+	JOIN hosts h ON hda.host_id = h.id
+WHERE
+ %s
+	`
+	var args []any
+	teamFilter := `h.team_id IS NULL`
+	if teamID != nil {
+		teamFilter = `h.team_id = ?`
+		args = append(args, teamID)
```

Comment:
That probably works as is? But just to avoid second-guessing later:
```suggestion
		args = append(args, *teamID)
```